doc/fa-dev-guide.md

```diff
@@ -119,6 +129,31 @@ def get_list(conn, options):
 	return outlets
 ```
 
+### reboot_cycle()
+***The reboot cycle method is not recommended as it might report success before the node is powered off.***
+- fence_ipmilan contains a minimal reboot_cycle() approach.
+- Add "method" to the device_opt list.
+- Update all_opt["method"]["help"] with a warning that it might report success before the node is powered off.
+- Add reboot_cycle function to fence_action() call in main().
+
+Example:
+```
+def reboot_cycle(_, options):
+        output = _run_command(options, "cycle")
+        return bool(re.search('chassis power control: cycle', str(output).lower()))
+...
+
+def main():
+...
+        device_opt = [ ..., "method" ]
+...
+        all_opt["method"]["help"] = "-m, --method=[method]          Method to fence (onoff|cycle) (Default: onoff)\n" \
+                                    "WARNING! This fence agent might report success before the node is powered off. " \
+                                    "You should use -m/method onoff if your fence device works correctly with that option."
```

Comment:
Maybe write `-m onoff` or `--method=onoff` instead of `-m/method onoff`

Reply:
All the parameters are listed like this.

Reply:
And the reason it is listed like this is probably to not be too wide on regular terminals (for longer parameters.

Example from fence_ipmilan -h:
   -S, --password-script=[script] Script to run to retrieve password

Reply:
> not be too wide 

I am not sure I would consider it as too wide. I have found "`-m onoff` or `--method=onoff`" to be more informative regarding the usage off this attribute (where I was confused), but if it contradict the other pattern which other parameters are listed then discard my comment.

Reply:
Yeah, but it would be inconsistent with the other parameters (which would be too wide if you expand them).